t/31-api_descriptions.t

```diff
@@ -2,13 +2,33 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 use Test::Most;
+use Test::MockModule;
 use Test::Warnings ':report_warnings';
+use Test::Output qw(combined_like);
 # no OpenQA::Test::TimeLimit for this trivial test
 
+use File::Temp qw(tempdir);
+use Mojo::File qw(path);
+
 use Mojo::Base 'Mojolicious', -signatures;
 
 use_ok('OpenQA::WebAPI::Description', qw(get_pod_from_controllers set_api_desc));
 my $app = Mojolicious->new;
 get_pod_from_controllers($app);
 
+# mock the parse_file
+my $mock = Test::MockModule->new('Pod::POM');
+$mock->redefine('parse_file' => undef);
```

Comment:
I think we don't run perlcritic on test files yet, otherwise it would have complained that they key is uselessly quoted:
```suggestion
$mock->redefine(parse_file => undef);
```

Reply:
@perlpunk can you check if we are missing perlcritic on tests? If we don't have that please create a ticket and/or draft PR for that

Reply:
Oh, I see